hera_mc/cm_transfer.py

```diff
@@ -136,8 +140,8 @@ def __initialization(args):
             print('Initialization for %s not found' % (table))
             use_table.remove(table)
         else:
-            if args.maindb:
-                if dbkey != args.maindb:
+            if maindb:
+                if dbkey != maindb:
```

Comment:
yeah, it took me a while to figure it out and it's another one of those times where it can have different types, either boolean or a string which is weird. but I agree, it should be changed in a different PR.